docs/user_guide/migrating_versions/index.rst

```diff
@@ -1,7 +1,7 @@
 .. _version_migration_guide:
 
-Version Migration Guide
-=======================
+Migrating from Prior Versions
```

Comment:
nit: lowercase p and v

Reply:
We haven't been entirely consistent on that sort of point, but I can change this one.